<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResourceDetailsWidget</class>
 <widget class="QWidget" name="ResourceDetailsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>455</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="widgetLayout">
   <item>
    <widget class="nx::vms::client::desktop::AsyncImageWidget" name="cameraPreviewWidget" native="true"/>
   </item>
   <item>
    <widget class="nx::vms::client::desktop::TextEditLabel" name="captionLabel"/>
   </item>
   <item>
    <widget class="QLabel" name="messageLabel">
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="warningContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>81</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>nx::vms::client::desktop::AsyncImageWidget</class>
   <extends>QWidget</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/async_image_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>nx::vms::client::desktop::TextEditLabel</class>
   <extends>QTextEdit</extends>
   <header location="global">nx/vms/client/desktop/common/widgets/text_edit_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
